src/components/NodeToolbar/AccessibilityEditor/WheelchairStatusEditor.js

```diff
@@ -217,7 +209,7 @@ class WheelchairStatusEditor extends React.Component<Props, State> {
           name="accessibility"
           selectedValue={wheelchairAccessibility}
           onChange={(newValue) => { this.setState({ wheelchairAccessibility: newValue }); }}
-          className={`${wheelchairAccessibility} ${valueIsDefined ? 'has-selection' : ''} radio-group`}
+          className={`${String(wheelchairAccessibility)} ${valueIsDefined ? 'has-selection' : ''} radio-group`}
```

Comment:
-0.5: `${String(wheelchairAccessibility)}` -> `String(null) = 'null'`  `String(undefined) = 'undefined')` 

Maybe `${wheelchairAccessibility || ''}` if your goal was to prevent class="undefined …"

Reply:
I was thinking of this case and thought having 'null' or 'undefined' as class there would make debugging easier if it happens… Or is this a bad idea? :)

Reply:
Yes, it's a bad idea. First it might of course select an unwanted class named 'undefined'. Second undefined is a valid value (at least according to the flow type, so it should be treated correctly. But maybe 'undefined' is the correct treatment then, if a selector for such a class would exist.